libtextsecure/storage/groups.js

```diff
@@ -65,11 +65,11 @@
         },
 
         removeNumber: function(groupId, number) {
-            var group = textsecure.storage.getEncrypted("group" + groupId);
+            var group = axolotl.api.storage.get("group" + groupId);
             if (group === undefined)
                 return undefined;
 
-            var me = textsecure.utils.unencodeNumber(textsecure.storage.getUnencrypted("number_id"))[0];
+            var me = textsecure.utils.unencodeNumber(textsecure.storage.getUnencrypted("number_id"))[0]();
```

Comment:
wat? if unencodeNumber is now returning a function as element 0, someone better tell all the other calls to unencodeNumber.